easybuild/easyconfigs/j/jhbuild/jhbuild-3.15.92-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'jhbuild'
+version = '3.15.92'
+
+homepage = 'https://wiki.gnome.org/action/show/Projects/Jhbuild'
+description = """JHBuild allows you to automatically download and compile “modules” (i.e. source code packages). 
+Modules are listed in “module set” files, which also include dependency information so that JHBuild can discover 
+what modules need to be built and in what order."""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+
+sources = ['%(version)s.tar.gz']
+source_urls = [('https://github.com/GNOME/jhbuild/archive/')]
+
+# jhbuild is python so it has a python dependency, but we want to give people freedom to use whatever python they
+# chose during a build process
+builddependencies = [
+    ('Python', '2.7.12', '-bare'),
```

Comment:
Well, wait, no, I got things mixed up, sort of.

If you specify you want to use the system Python, that'll be enforced at build time (sort of), but it has no effect on the generated module, so the runtime dep on the system Python is implicit there; which means you can override it by loading Python as a module somehow, either manually or via a dep of something else.

Sorry for the confusion...